distribution/docs/src/main/resources/content/_using/uploading.adoc

```diff
@@ -0,0 +1,45 @@
+:title: Uploading from ${catalog-ui}
```

Comment:
❓ Should this file be renamed to reflect that this is uploading through the `Catalog UI` rather than uploading through something like the Karaf command line?

Reply:
sure